src/test/run-make-fulldeps/rustdoc-map-file/validate_json.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import json
+
+
+def find_redirect_map_file(folder, errors):
+    for entry in os.listdir(folder):
+        full_path = os.path.join(folder, entry)
+        if os.path.isdir(full_path):
+            if find_redirect_map_file(full_path, errors):
+                return True
```

Comment:
Rather than using recursion, you can use `os.walk(folder)`: https://docs.python.org/3/library/os.html#os.walk. There's an example here: https://stackoverflow.com/questions/18860679/searching-a-directory-for-folders-and-files-using-python.

Reply:
I'm too used to do stuff "by hand"... Thanks!